core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -152,13 +152,37 @@
    *
    * @param skipNegotiation @{code true} if there is a priori knowledge that the endpoint supports
    *                        plaintext, {@code false} if plaintext use must be negotiated.
+   * @deprecated Use {@link #usePlaintext()} instead.
    *
    * @throws UnsupportedOperationException if plaintext mode is not supported.
    * @return this
    * @since 1.0.0
    */
+  @Deprecated
```

Comment:
Keep the ExperimentalApi annotation, so that users know it can actually be removed.

Reply:
Done.